### PR TITLE
feat: add module system

### PR DESCRIPTION
**Pull request by JulianFeinauer**
OMAP can now be started via a config system.
More informations can be found in the README.

### CODE REVIEW

File: omap.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import sys
+
+from omap.modules import modules
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "omap.launcher.settings")
+    # Here we set the PORTAL_MODULES env varible, if not set
+    os.environ.setdefault("PORTAL_MODULES", "")
```
Comment (NiklasMerz) [inline omap.py]: Why `PORTAL_MODULES`? Something OMAP would fit better right?

File: omap/launcher/settings.py [hunk 2]
```diff
@@ -41,9 +41,9 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "tailwind",
-    "omap.core",
-    "omap.assets",
+    # "tailwind",
+    # "omap.core",
+    # "omap.assets",
```
Comment (NiklasMerz) [inline omap/launcher/settings.py]: Just remove it instead of commenting?

File: omap/launcher/urls.py [hunk 3]
```diff
@@ -1,7 +1,42 @@
...
 from django.conf.urls import url, include
+from django.conf.urls.static import static
 from django.contrib import admin
+from django.urls import path
+
+import omap.modules.modules
 
 urlpatterns = [
-    url("", include("omap.core.urls")),
+    # url("", include("omap.core.urls")),
```
Comment (NiklasMerz) [inline omap/launcher/urls.py]: Another comment

File: omap/modules/modules.py [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+def modules():
+    return _modules
+
+
+class ModuleConfig:
+    pass
+
+
+class PortalModule(object):
```
Comment (NiklasMerz) [inline omap/modules/modules.py]: Rename to OMAP to distinguish OMAP and Portal?

**Review by NiklasMerz — COMMENTED** [review]
Just some naming things

**Comment by JulianFeinauer** [comment]
@NiklasMerz all fixed :)

**Pull request merged**